Resources/Prototypes/Entities/Objects/Specific/chemistry-vials.yml

```diff
@@ -0,0 +1,92 @@
+# TODO: Have a seperate lid state that can be popped on and off (not as a seperate item) that changes whether the vial can be spilt or not
+
+- type: entity
+  name: vial
+  parent: BaseItem
+  id: BaseChemistryEmptyVial
+  abstract: true
```

Comment:
This line marks the prototype as being primarily a template for other prototypes. If you remove it you should be able to spawn it in.